larpandoracontent/LArVertex/MvaVertexSelectionAlgorithm.cc

```diff
@@ -680,19 +687,30 @@ void MvaVertexSelectionAlgorithm<T>::GetBestVertex(const VertexVector &vertexVec
     const MCParticleList *pMCParticleList(nullptr);
     PANDORA_THROW_RESULT_IF(STATUS_CODE_SUCCESS, !=, PandoraContentApi::GetList(*this, m_mcParticleListName, pMCParticleList));
 
-    // Obtain vector: true neutrinos
-    MCParticleVector mcNeutrinoVector;
-    LArMCParticleHelper::GetTrueNeutrinos(pMCParticleList, mcNeutrinoVector);
+    // Obtain vector: true targets
+    MCParticleVector mcTargetVector;
+    LArMCParticleHelper::GetTrueNeutrinos(pMCParticleList, mcTargetVector);
```

Comment:
Should this be removed, as it's also in the else block, below (which seems to be the correct location)?

Reply:
That's a bad one!  Good spot.